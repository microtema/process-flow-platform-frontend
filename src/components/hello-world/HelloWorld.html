<div class="hello-world">
    <h1>{{ msg }}</h1>
</div>
