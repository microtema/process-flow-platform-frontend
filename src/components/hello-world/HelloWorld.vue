<template src="./HelloWorld.html" />
<script src="./HelloWorld.component.ts" lang="ts" />
<style src="./HelloWorld.scss" scoped lang="scss" />
