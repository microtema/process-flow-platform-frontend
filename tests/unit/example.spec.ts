import { shallowMount } from '@vue/test-utils'
import HelloWorldComponent from '@/components/hello-world/HelloWorld.component'
import HelloWorld from '@/components/hello-world/HelloWorld.vue'

describe('HelloWorld.vue', () => {
  it('verify html', () => {
    const msg = 'new message'

    const wrapper = shallowMount(HelloWorld, {
      propsData: { msg }
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
