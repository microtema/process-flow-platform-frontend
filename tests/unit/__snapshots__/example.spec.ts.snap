// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue verify html 1`] = `
<div class="hello-world" msg="new message">
  <h1></h1>
</div>
`;
